client/packages/purchasing/src/purchase_order/api/hooks/usePurchaseOrderLine.ts

```diff
@@ -61,12 +63,20 @@ export function usePurchaseOrderLine(id?: string) {
         100
       : 0;
 
+  // Number of packs is not in the DB, so we derived it from the draft
+  const initialNumberOfPacks = data?.nodes[0]?.requestedNumberOfUnits
+    ? (data?.nodes[0]?.adjustedNumberOfUnits ??
+        data?.nodes[0]?.requestedNumberOfUnits ??
+        0) / (data?.nodes[0]?.requestedPackSize ?? 1)
+    : 0;
```

Comment:
So we're saying use the "adjusted" value if there is one, otherwise use the "requested" value?

In which case, isn't this a bit clearer?
```ts
  const initialNumberOfPacks =
    (data?.nodes[0]?.adjustedNumberOfUnits ??
      data?.nodes[0]?.requestedNumberOfUnits ??
      0) / (data?.nodes[0]?.requestedPackSize);
```

Or, to be clearer still:
```ts
  const adjustedUnits = data?.nodes[0]?.adjustedNumberOfUnits;
  const requestedUnits = data?.nodes[0]?.requestedNumberOfUnits ?? 0;
  const requestedPackSize = data?.nodes[0]?.requestedPackSize ?? 1;

  const initialNumberOfPacks =
    (adjustedUnits ?? requestedUnits) / requestedPackSize;
```

(I know, I probably should have done this for the `initialDiscountPercentage` too ;)